jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by replacing references (such as "$ref") with an
+                        embedded version of the referenced Schema Resource.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource (and the document's root) MUST identify itself
+                        with an absolute URI using the "$id" keyword, and SHOULD make use of the "$schema"
+                        keyword to identify the dialect it is using, in the root of the schema resource.
+                    </t>
+                    <t>
+                        When a reference applicator is dereferenced as part of the bundling process, the Schema Resource
+                        MUST NOT be embedded by replacing the schema object from which it was referenced, or by wrapping
+                        in other applicator keywords. In stead, a bundled Schema Resource MUST be located in `$defs`,
+                        where the key is the `$id` of the Schema Resource, which MUST then be referenced using `$ref`.
+                    </t>
+                </section>
+                <section title="Validating">
+                    <t>
+                        Given a Compound Schema Document may have embedded resources which idenfiy as using different
+                        dialects, it is NOT RECOMMENDED that these documents are validated by applying a meta-schema
+                        to the Compound Schema Document as an instance. It is RECOMMENDED that an alternate
+                        validation process be provided in order to validate Compound Schema Documents
```

Comment:
Does this need to be an alternate for _compound_ schema documents, or just an alternate for schema documents (schemas against meta-schemas) in general?  You should always be able to tell whether a given schema is compound, so if you know your instance is a schema that is enough.

We don't want to encourage two different meta-schema functions, because if someone validating a non-bundled schema with the "regular" meta-schema validation function, but then that schema got changed to a bundled schema, it would now require a code change to be handled correctly.  We _never_ want to require a code change to keep validation working.

Reply:
The consensus in the associated issue was that a compound document would be transport only, in that you can't validate it by applying a meta schema to it as an instance.
Check the issue for the reasoning and justification.

Reply:
@Relequestual I think @handrews was just concerned that this could be misunderstood to mean that you need a separate process to validate a schema than you do to validate a compound schema. The process for validating schema documents *is* the process for validating compound schema documents.

I think the suggestion would be to do something like ...

> It is RECOMMENDED that an alternate validation process be provided in order to validate ~Compound~ Schema Documents

The "Compound" part is implied because all schemas are potentially compound.

Reply:
@Relequestual yes, what @jdesrosiers said.

Reply:
Ok, we are on the same page now. I don't disagree. 
But, I'm now not sure about a few things. I'm saying the following having not reviewed the issue again (because this is a quick comment in a few mins spare time).

I thought in the issue we said it wouldn't be possible to identify a compound schema document vs a normal schema document, although that may have been in context of being an instance.

I'm going to HAVE to review the issue discussion again. It's fine =]

Reply:
> I thought in the issue we said it wouldn't be possible to identify a compound schema document vs a normal schema document, although that may have been in context of being an instance.

Yes, that was in the context of being an instance. It's not possible to identify an instance as a schema (compound or otherwise).